regions/io/core.py

```diff
@@ -29,12 +65,118 @@ def to_regions(self):
                 msg = 'Skipping elliptical annulus {}'.format(shape)
                 warn(msg, DS9RegionParserWarning)
                 continue
+            if shape.region_type in ['box', 'symbol', 'text'] and shape.format_type == 'CRTF':
+                msg = 'Skipping {} {}'.format(shape.region_type, shape)
```

Comment:
Add a message here: `Skipping ... because there is no equivalent representation for this region type in CRTF.  See the documentation at (put a URL here) to see which CRTF regions are allowed.`

Reply:
These regions are skipped for just now, will solve this in the next PR